### PR TITLE
Update docs of Harbor chart

### PR DESCRIPTION
**Pull request by ywk253100**
Update Readme and ha doc of Harbor chart

Signed-off-by: Wenkai Yin <yinw@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,31 +178,55 @@ The following tables lists the configurable parameters of the Harbor chart and t
...
 | **Clair** |
-| `clair.enabled` | Enable Clair? | `true` |
+| `clair.enabled` | Enable Clair | `true` |
 | `clair.image.repository` | Repository for clair image | `goharbor/clair-photon` |
 | `clair.image.tag` | Tag for clair image | `dev`
 | `clair.replicas` | The replica count | `1` |
-| `clair.proxy.enabled` | Enable the use of http(s) proxy | `false` |
-| `clair.proxy.http_proxy` | URL of the http proxy |
-| `clair.proxy.https_proxy` | URL of the https proxy |
+| `clair.httpProxy` | URL of the http proxy ||
```
Comment (reasonerjt) [inline README.md]: add `URL of the http proxy, via which Clair will download the vulnerability data from the internet`

File: README.md [hunk 2]
```diff
@@ -23,45 +20,115 @@ Checkout the branch.
...
-| **Harbor** |
-| `persistence.enabled`     | Persistent data | `true` |
-| `persistence.resourcePolicy`     | Setting it to "keep" to avoid removing PVCs during a helm delete operation. Leaving it empty will delete PVCs after the chart deleted | `keep` |
-| `externalURL`       | Ther external URL for Harbor core service | `https://core.harbor.domain` |
-| `harborAdminPassword`  | The password of system admin | `Harbor12345` |
-| `secretkey` | The key used for encryption. Must be a string of 16 chars | `not-a-secure-key` |
+| **Expose** |
+|`expose.type`|The way how to expose the service: `ingress`, `clusterIP` or `nodePort`|`ingress`|
+|`expose.tls.enabled`|Enable the tls or not|`true`|
+|`expose.tls.secretName`|Fill the name of secret if you want to use your own TLS certificate and private key. The secret must contain keys named `tls.crt` and `tls.key` that contain the certificate and private key to use for TLS. The certificate and private key will be generated automatically if leaving it empty||
```
Comment (reasonerjt) [inline README.md]: `...... if leaving it empty` -> `...... if it is not set`

File: README.md [hunk 3]
```diff
@@ -23,45 +20,115 @@ Checkout the branch.
...
+|`persistence.persistentVolumeClaim.chartmuseum.subPath`     | The sub path used in the volume | `chartmuseum` |
+|`persistence.persistentVolumeClaim.chartmuseum.accessMode`     | The access mode of the volume | `ReadWriteMany` |
+|`persistence.persistentVolumeClaim.chartmuseum.size`     | The size of the volume | `5Gi` |
+|`persistence.persistentVolumeClaim.jobservice.existingClaim`     | Use the existing PVC which must be created manually before bound | |
+|`persistence.persistentVolumeClaim.jobservice.storageClass`     | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used(the default). Set it to `-` to disable dynamic provisioning | |
+|`persistence.persistentVolumeClaim.jobservice.subPath`     | The sub path used in the volume | `jobservice` |
+|`persistence.persistentVolumeClaim.jobservice.accessMode`     | The access mode of the volume | `ReadWriteMany` |
+|`persistence.persistentVolumeClaim.jobservice.size`     | The size of the volume | `1Gi` |
+|`persistence.persistentVolumeClaim.database.existingClaim`     | Use the existing PVC which must be created manually before bound | |
```
Comment (reasonerjt) [inline README.md]: Shall we document if user chooses to use external DB/Redis the PVC for DB/Redis will be ignored?

File: README.md [hunk 4]
```diff
@@ -23,45 +20,115 @@ Checkout the branch.
...
-| `ingress.hosts.core` | The host of Harbor core service in ingress rule | `core.harbor.domain` |
-| `ingress.hosts.notary` | The host of Harbor notary service in ingress rule | `notary.harbor.domain` |
-| `ingress.annotations` | The annotations used in ingress | `true` |
-| `ingress.tls.enabled` | Enable TLS | `true` |
-| `ingress.tls.secretName` | Fill the secretName if you want to use the certificate of yourself when Harbor serves with HTTPS. A certificate will be generated automatically by the chart if leave it empty |
-| `ingress.tls.notarySecretName` | Fill the notarySecretName if you want to use the certificate of yourself when Notary serves with HTTPS. if left empty, it uses the `ingress.tls.secretName` value |
+| `logLevel` | The log level | `debug` |
+| `harborAdminPassword`  | The initial password of Harbor admin. Change it from portal after launching Harbor | `Harbor12345` |
+| `secretkey` | The key used for encryption. Must be a string of 16 chars | `not-a-secure-key` |
+| **Nginx** |
```
Comment (reasonerjt) [inline README.md]: Shall we document if the ingress is chosen in `expose` chunk the Nginx will not be created?

File: README.md [hunk 5]
```diff
@@ -1,19 +1,16 @@
...
 
 ## Introduction
-
 This [Helm](https://github.com/kubernetes/helm) chart installs [Harbor](https://github.com/goharbor/harbor) in a Kubernetes cluster. Welcome to [contribute](CONTRIBUTING.md) to Helm Chart for Harbor.
 
 ## Prerequisites
-
-- Kubernetes cluster 1.10+ with Beta APIs enabled
-- Kubernetes Ingress Controller is enabled
+- Kubernetes cluster 1.10+
```
Comment (reasonerjt) [inline README.md]: Shall we document if user chooses to use external DB/Redis the PVC settings for DB/Redis will be ignored?

**Comment by ywk253100** [comment]
@reasonerjt Updated.

**Pull request merged**